lib/config/presets/internal/workarounds.ts

```diff
@@ -7,8 +7,8 @@ export const presets: Record<string, Preset> = {
     ],
```

Comment:
Do we need/want a config migration?

Reply:
Don't think I've run into that feature yet. How would I add one?

Reply:
Sometimes when we change a config option, we add a migration to this file: https://github.com/renovatebot/renovate/blob/main/lib/config/migration.ts

The maintainers will know if we need a config migration. I think it's best if you wait a bit, until we have a response from them, before you start doing work that might not be needed.

Reply:
Not sure we need it as long as we're leaving the old, deprecated preset there